plugins/kubernetes/app/models/watchers/cluster_pod_error_watcher.rb

```diff
@@ -0,0 +1,35 @@
+module Watchers
+  class ClusterPodErrorWatcher < BaseClusterWatcher
+    include Celluloid::Notifications
+
+    def initialize(client)
+      @client = client
+      super(@client.watch_events(field_selector: 'involvedObject.kind=Pod'))
+    end
+
+    private
+
+    def handle_notice(notice)
+      if error_notice?(notice.object.reason)
+        project_name = project_name(notice.object.involvedObject)
+        publish(project_name, notice) if project_name
```

Comment:
When I submit my PR, after merging with this changes, we must double check that I have changed this to `project_id` (which was added to the metadata labels as well).